tests/google/stubs.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import Mock
+
+
+def google_request_stub(resources, resource_type_label):
+    mock_execute = Mock()
+    mock_execute.return_value = {resource_type_label: resources}
```

Comment:
This can be simplified:
```suggestion
    mock_execute = Mock(return_value={resource_type_label: resources})
```